DOCUMENTATION.md

```diff
@@ -330,20 +330,23 @@ If the request is not successful, the client returns a `Notify.Exceptions.Notify
 
 ## Send a file by email
 
-Send files without the need for email attachments.
+To send a file by email, add a placeholder to the template then upload a file. The placeholder will contain a secure link to download the file.
 
-This is an invitation-only feature. [Contact the GOV.UK Notify team](https://www.notifications.service.gov.uk/support/ask-question-give-feedback) to enable this function for your service.
+The links are unique and unguessable. GOV.UK Notify cannot access or decrypt your file.
 
-To send a file by email, add a placeholder field to the template and then upload a file. The placeholder field will contain a secure link to download the file.
+#### Add contact details to the file download page
```

Comment:
This heading and the `Add a placeholder to the template` heading should be `H3`.